ksp_plugin/vessel.hpp

```diff
@@ -134,12 +134,15 @@ class Vessel {
       not_null<Ephemeris<Barycentric>*> const ephemeris,
       not_null<Celestial const*> const parent);
 
-  void set_pile_up(ContainerIterator<std::list<PileUp>> pile_up);
-  std::experimental::optional<ContainerIterator<std::list<PileUp>>> pile_up()
-      const;
+  void set_containing_pile_up(ContainerIterator<std::list<PileUp>> pile_up);
```

Comment:
Wondering if these functions (and the member variable) should be plural, i.e., `set_containing_pile_ups` (where "ups" is obviously the plural of "up").